sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ModelBridgeInternal.java

```diff
@@ -540,6 +540,11 @@ public static Integer getMaxItemCountFromQueryRequestOptions(CosmosQueryRequestO
         return options.getMaxItemCount();
     }
 
+    @Warning(value = INTERNAL_USE_ONLY_WARNING)
```

Comment:
Please no new members here - instead use ImplementationBridgeHelper pattern

Reply:
Fixed the code.